app.go

```diff
@@ -591,6 +606,12 @@ func New(config ...Config) *App {
 	if app.config.JSONDecoder == nil {
 		app.config.JSONDecoder = json.Unmarshal
 	}
+	if app.config.MsgPackEncoder == nil {
+		app.config.MsgPackEncoder = msgpack.Marshal
+	}
+	if app.config.MsgPackDecoder == nil {
+		app.config.MsgPackDecoder = msgpack.Unmarshal
```

Comment:
This should use utils.MsgPackUnmarshal to avoid the import at the top

Reply:
Same reason, we can't use that way since utils.MsgPackUnmarshal is a type.